odoo/addons/base/models/ir_actions.py

```diff
@@ -637,17 +639,27 @@ def run(self):
                 try:
                     records.check_access_rule('write')
                 except AccessError:
-                    _logger.warning("Forbidden server action %r executed while the user %s does not have access to %s.",
-                        action.name, self.env.user.login, records,
+                    _logger.warning("%s Forbidden server action %r executed while the user %s does not"
+                                    " have access to %s.", LogType.SERVER_ACTION_FORBIDDEN, action.name,
+                                    self.env.user.login, records,
                     )
                     raise
 
+            def log():
+                # Avoid logging CRON that are already logged
+                if action.usage != "ir_cron":
+                    # Log must be before the runner. In case an exception is raised in the SA.
+                    _logger.info("%s %r (#%d) has been run by user %r (#%d).", LogType.SERVER_ACTION_RUN,
+                                 self.display_name, self.id, self.env.user.display_name, self.env.user.id)
+
             runner, multi = action._get_runner()
             if runner and multi:
+                log()
```

Comment:
no need to repeat, you can simply
```suggestion
            if runner and action.usage != "ir_cron":
                _logger...
            if runner and multi:
```